scout/server/blueprints/variants/controllers.py

```diff
@@ -581,11 +603,25 @@ def cancer_variants(store, institute_id, case_name, variants_query, variant_coun
 
     variant_res = variants_query.skip(skip_count).limit(per_page)
 
+    # get gene panel lookup table
+    gene_panel_lookup = store.gene_to_panels(case_obj)  # build variant object
```

Comment:
Hm, isn't this still relevant though:
https://github.com/Clinical-Genomics/scout/pull/2395#discussion_r615918310
It is a very good start as is, but I imagine the manual gene symbol search box and HPO / dynamic panel users will be confused once in a while when their searched variant is shown with a different gene name?